frontend/src/utils/hooks/useDownloadCardImage.tsx

```diff
@@ -20,11 +27,18 @@ const BOTTOM_PADDING = 120
 const URL_FONT_SIZE = 14
 const URL_FONT_STYLE = '"Inter",sans-serif'
 
+export type HiddenElements =
```

Comment:
this is great! What do you think about using something like `ElementHashIdsHiddenOnScreenshot` ? I know it's super long but i'd like to use `HashId` in the name so it's clear it's a hashed id (and not like a ref to an html el or something) and I'd like it to be clear to future devs the elements _can_ be hidden, but aren't normally hidden

Reply:
Sounds good to me ☺️